docs/getting-started.adoc

```diff
@@ -0,0 +1,62 @@
+= Get Started with Asciidoctor
```

Comment:
I think there's too much content in this page. Maybe we should split this page "Getting started" and "Ecosystem" ?